src/main/java/org/opensearch/neuralsearch/query/HybridQueryBuilder.java

```diff
@@ -295,4 +297,16 @@ private Collection<Query> toQueries(Collection<QueryBuilder> queryBuilders, Quer
         }).filter(Objects::nonNull).collect(Collectors.toList());
         return queries;
     }
+
+    /**
+     * visit method to parse the HybridQueryBuilder by a visitor
+     */
+    @Override
+    public void visit(QueryBuilderVisitor visitor) {
+        visitor.accept(this);
+        QueryBuilderVisitor subVisitor = visitor.getChildVisitor(Occur.MUST);
```

Comment:
Could you tell why only `MUST` but not others like `SHOULD` and `MUST_NOT`?

Reply:
Could you add that in comment that The value can be anything because `NeuralSearchQueryVisitor.getChildVisitor` return `this` regardless the value?

Reply:
Sure